helix-core/src/test/java/org/apache/helix/monitoring/mbeans/TestDropResourceMetricsReset.java

```diff
@@ -34,11 +34,15 @@
 import org.apache.helix.integration.manager.MockParticipantManager;
 import org.apache.helix.model.IdealState.RebalanceMode;
 import org.apache.helix.tools.ClusterSetup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class TestDropResourceMetricsReset extends ZkUnitTestBase {
+  private static final Logger LOG = LoggerFactory.getLogger(TestDropResourceMetricsReset.class);
+
```

Comment:
Also, I recommend avoiding force-pushes because that wipes out all commit / comment trails. You can just create another commit on top of the previous commit.